utils/mods/legacy_combat.lua

```diff
@@ -267,7 +267,7 @@ function TryCriticalHit(e)
 		return e;
 	end
 
-	if (self:IsPet() and self:GetOwner().valid and self:GetOwner():IsBot()) then
+	if (self:IsPet() and self:GetOwner().valid and self:GetOwner():is_bot()) then
```

Comment:
`IsBot()` is valid, `is_bot()` is not.

Reply:
Yeah, this got caught up in a renaming unfortunately of the member attribute. I'll fix these up.